docs/api/native-image.md

```diff
@@ -192,14 +193,14 @@ Creates a new `NativeImage` instance from `dataURL`.
 
 Returns `NativeImage`
 
-Creates a new `NativeImage` instance from the NSImage that maps to the
-given image name. See [`System Icons`](https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/system-icons/)
-for a list of possible values.
+Creates a new `NativeImage` instance from the `NSImage` that maps to the
+given image name. See Apple's [`System Icons`](https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/system-icons/)
```

Comment:
Unfortunately this link 404s. [Here's what it used to look like](https://web.archive.org/web/20220128180553/https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/system-images/). Based on the code for `createFromNamedImage`, perhaps this is a good link to replace it with? https://developer.apple.com/documentation/appkit/nsimagename#2901388

Reply:
```suggestion
given image name. See Apple's [`NSImageName`](https://developer.apple.com/documentation/appkit/nsimagename#2901388)
```